blazor/gantt-chart/scrolling.md

```diff
@@ -307,4 +307,305 @@ RowIndex : <input @bind-value = "@RowIndex" />
+    </GanttTaskFields>
+    <GanttSplitterSettings Position="60%"></GanttSplitterSettings>
+</SfGantt>
+<style>
+    .e-grid .e-frozenheader > .e-table,
+    .e-grid .e-frozencontent > .e-table,
+    .e-grid .e-frozencontent .e-virtualtable > .e-table,
+    .e-grid .e-frozenheader .e-virtualtable > .e-table {
+        border-right-color: blue;
+    }
+
+    .e-grid .e-frozenhdrcont .e-headercontent > .e-table,
+    .e-grid .e-frozenhdrcont .e-frozenheader > .e-table,
+    .e-grid .e-frozenhdrcont .e-movableheader > .e-table,
+    .e-grid .e-frozenhdrcont .e-headercontent .e-virtualtable > .e-table {
+        border-bottom-color: blue;
+    }
+</style>
+@code {
+    private List<TaskData> TaskCollection { get; set; }
+    protected override void OnInitialized()
+    {
+        this.TaskCollection = GetTaskCollection();
+    }
+
+    public class TaskData
+    {
+        public int TaskId { get; set; }
+        public string TaskName { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public string Duration { get; set; }
+        public int Progress { get; set; }
+        public int? ParentId { get; set; }
+    }
+
+    public static List<TaskData> GetTaskCollection()
+    {
+        List<TaskData> Tasks = new List<TaskData>() {
+        new TaskData() { TaskId = 1, TaskName = "Project initiation", StartDate = new DateTime(2023, 02, 05), EndDate = new DateTime(2022, 02, 21) },
+        new TaskData() { TaskId = 2, TaskName = "Identify Site location", StartDate = new DateTime(2023, 02, 05), Duration = "0", Progress = 30, ParentId = 1 },
+        new TaskData() { TaskId = 3, TaskName = "Perform soil test", StartDate = new DateTime(2023, 02, 05), Duration = "4", Progress = 40, ParentId = 1 },
+        new TaskData() { TaskId = 4, TaskName = "Soil test approval", StartDate = new DateTime(2023, 02, 05), Duration = "0", Progress = 30, ParentId = 1 },
+        new TaskData() { TaskId = 5, TaskName = "Project estimation", StartDate = new DateTime(2023, 02, 06), EndDate = new DateTime(2023, 02, 21) },
+        new TaskData() { TaskId = 6, TaskName = "Develop floor plan for estimation", StartDate = new DateTime(2023, 02, 06), Duration = "3", Progress = 30, ParentId = 5 },
+        new TaskData() { TaskId = 7, TaskName = "List materials", StartDate = new DateTime(2023, 02, 06), Duration = "3", Progress = 40, ParentId = 5 },
+        new TaskData() { TaskId = 8, TaskName = "Estimation approval", StartDate = new DateTime(2023, 02, 06), Duration = "0", Progress = 30, ParentId = 5 }
+    };
+        return Tasks;
+    }
+}
+```
+![Changing freeze line color inBlazor Gantt Chart](./images/blazor_Gantt_chart_freeze_line_color.gif)
+
+### Limitations
+
+- The `Freeze` and `FrozenColumns` properties cannot be used together as they are incompatible.
+
+- The `IsFrozen` property cannot be changed dynamically. Once the column is frozen, it remains frozen until the chart is reloaded.
```

Comment:
Need clarification

Reply:
We have achieve this in sample level. Limitation removed